src/iterateJsdoc.js

```diff
@@ -464,6 +481,16 @@ const iterate = (
   const indent = sourceLine.charAt(0).repeat(jsdocNode.loc.start.column);
   const jsdoc = parseComment(jsdocNode, indent, !ruleConfig.noTrim);
   const report = makeReport(context, jsdocNode);
+  const userConfig = context.options.length > 0 ? _.cloneDeep(context.options) : [{}];
+
+  if (ruleConfig.meta.schema) {
+    new Ajv({useDefaults: true}).compile(
+      {
+        items: ruleConfig.meta.schema,
+        type: 'array',
+      },
+    )(userConfig);
+  }
```

Comment:
Is this block for the intent of validating the schema? If so, that shouldn't be necessary as eslint handles that for us.

Reply:
I have noticed, that `eslint` doesn't apply defaults to the `options`-array unless...
- Either a `default` is declared at the root of the meta-data:
```js
meta: {
  fixable: 'code',
  schema: [
    {
      default: {
        allowTrailingParamDocs: false // Duplicate declaration of the default  value for `allowTrailingParamDocs`
      },
      additionalProperties: false,
      properties: {
        allowExtraTrailingParamDocs: {
          type: 'boolean',
          default: false // Duplicate declaration of the default value for `allowTrailingParamDocs`
        },
      },
      type: 'object',
    },
  ],
  type: 'suggestion',
}
```

Or by using the bunch of code that I added here (which is providing an array with an empty object `[{}]` to ajv.)

Reply:
While that is a sound approach, in order to avoid extra dependencies (and in case eslint ever fixes this and then we can drop the handling entirely), I've added 49f106c1b43331be63e1886f8a50e0b5079a0247 which changes to just manually check the schema defaults of interest here.

Reply:
Awesome thank you very much 😄